app/presenters/transaction_presenter.rb

```diff
@@ -48,6 +50,12 @@ def start_button_text
     end
   end
 
+  def show_country_notice?
+    return false if Date.today < (Date.new(2020,04,01) + 14.days)
+
+    @content_item["content_id"] == "89edffd2-3046-40bd-810c-cc1a13c05b6a"
```

Comment:
I wonder, given this is a short term experiment, whether it's best to use the base_path, having this makes it a lot easier to know which page it's used on.

For something longer term, content_id is best though so I can't fault your decision to use it.

Reply:
Yes definitely, I looked at similar implementations where we needed it for a specific content item and we have always used content id rather than base path